Makefile

```diff
@@ -287,6 +287,7 @@ install: $(build_bindir)/stringreplace doc/_build/html
 	$(INSTALL_M) $(build_bindir)/julia* $(DESTDIR)$(bindir)/
 ifeq ($(OS),WINNT)
 	-$(INSTALL_M) $(build_bindir)/*.dll $(DESTDIR)$(bindir)/
+	-$(INSTALL_M) $(build_libdir)/*.dll.a $(DESTDIR)$(libdir)/
```

Comment:
```
$ du -sh *.dll.a
 52K	libarpack.dll.a
 52K	libfftw3.dll.a
 52K	libfftw3f.dll.a
224K	libmpfr.dll.a
 68K	libpcre2-8.dll.a
4.0K	libpcre2-posix.dll.a
224K	libuv.dll.a
```
this will cause a few extra unnecessary files to be copied over. while they are all pretty small, i think would be nicer to avoid that and just copy `libjulia.dll.a` and `libjulia-debug.dll.a` explicitly.